packages/components/molecules/f-mega-modal/README.md

```diff
@@ -108,22 +108,23 @@ The following rudimentary example can be used as a guide for implementing this c
-| `title` | `String` | `''` | When set, will add a title to the top of the component. If you need to define a custom heading, ignore this prop.  |
-| `titleHtmlTag` | `String` | `h3` | Sets the tag for the component's title.<br><br>Allowed values are `h1`, `h2`, `h3`, and `h4` |
+| Prop                   | Type      | Default         | Description                                                                                                                                                       |
+|------------------------|-----------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `is-open`              | `Boolean` | `false`         | Sets the modal to open or closed state.                                                                                                                           |
+| `is-narrow`            | `Boolean` | `false`         | Use the narrow visual style.                                                                                                                                      |
+| `is-wide`              | `Boolean` | `false`         | Use the wide visual style.                                                                                                                                        |
+| `is-flush`             | `Boolean` | `false`         | Removes passing around the modal content.                                                                                                                         |
+| `is-full-height`       | `Boolean` | `false`         | Sets the modal content to full height of the screen.<br/><br/><blockquote>Note this only applies to small screen devices.</blockquote>                            |
+| `is-scrollable`        | `Boolean` | `false`         | Makes the modal content scrollable.                                                                                                                               |
+| `is-close-fixed`       | `Boolean` | `false`         | Sets the modal close button position to `fixed`.                                                                                                                  |
+| `is-positioned-bottom` | `Boolean` | `false`         | Sets the modal position to the bottom of the viewport for all screen widths.                                                                                      |
+| `has-overlay`          | `Boolean` | `true`          | Controls whether or not to display an overlay behind the modal.                                                                                                   |
+| `close-button-style`   | `String`  | `close-small-icon`          | Controls the style of the button. Accepts `close-small-icon` & `chevron-left-icon`. If you pass an empty string '', no styles for the close button will be shown. |
```

Comment:
Sorry just another nitpick from me, I think if we're going to have the prop called `closeButtonStyle`, we would want to accept clearer, simpler strings like "cross" or "chevron", and then handle the icons within.

Alternatively, we could change the name of this prop to `closeButtonIcon` which indicates that we're passing in the icon name. This is definitely the simpler option to implement.

Reply:
I'd probably go with the first option too if possible, just as it relies less on the user needing to know the icon naming.

Reply:
Agreed. Fixed now hopefully.